src/components/molecules/Visualizer/hooks.ts

```diff
@@ -75,6 +77,7 @@ export default ({
 }) => {
   const engineRef = useRef<EngineRef>(null);
   const [overriddenSceneProperty, overrideSceneProperty] = useOverriddenProperty(sceneProperty);
+  const [, zoomToLayer] = useZoomedLayerId();
```

Comment:
So somewhere else we are doing `const [zoomedLayerId] = useZoomedLayerId();`, and then here you are doing `const [, zoomToLayer] = useZoomedLayerId();`
I feel like it makes more sense to declare them in the same place (not sure here or where `zoomedLayerId` is currently declared is better. Please look at the code and decide which is better)

Reply:
Please see above comment about import restrictions.